packages/react-examples/src/react-charting/LineChart/LineChart.Basic.Example.tsx

```diff
@@ -186,6 +186,7 @@ export class LineChartBasicExample extends React.Component<{}, ILineChartBasicSt
             margins={margins}
             xAxisTickCount={10}
             allowMultipleShapesForPoints={this.state.allowMultipleShapes}
+            enablePerfOptimization={true}
```

Comment:
Did you give the PR for testing after enabling this flag for all chart types.

Reply:
No, earlier i have tested it locally for all the variants.